test.js

```diff
@@ -1,22 +1,16 @@
-'use strict';
-var childProcess = require('child_process');
-var test = require('ava');
-var taskkill = require('./');
+import childProcess from 'child_process';
+import test from 'ava';
+import fn from './';
```

Comment:
I want something consistent (even between projects) and short since it's usually duplicated a lot in tests, although not here.